packages/react-image/src/components/Image/useImage.ts

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+// @ts-ignore Typings require esModuleInterop
+import { objectFitImages } from 'object-fit-images';
+import { makeMergeProps, resolveShorthandProps } from '@fluentui/react-utils';
+import { useMergedRefs } from '@fluentui/react-hooks';
+import { ImageProps, ImageState } from './Image.types';
+
+const mergeProps = makeMergeProps();
+
+const isFitSupported = (function() {
+  const testImg = new Image();
+
+  return 'object-fit' in testImg.style && 'object-position' in testImg.style;
+})();
+
+/**
+ * Given user props, returns state and render function for an Image.
+ */
+export const useImage = (props: ImageProps, ref: React.Ref<HTMLElement>, defaultProps?: ImageProps): ImageState => {
+  const resolvedRef = useMergedRefs(ref, React.useRef());
+  const state = mergeProps(
+    {
+      ref: resolvedRef,
+      as: 'img',
+    },
+    defaultProps,
+    resolveShorthandProps(props, []),
+  );
+
+  const imageRef = React.useRef<HTMLElement>(null);
+
+  React.useEffect(() => {
+    if (!isFitSupported) {
+      objectFitImages(imageRef.current);
+    }
+  }, []);
+
+  state['aria-hidden'] = state.alt || state['aria-label'] ? undefined : 'true';
+  state.imageRef = imageRef;
```

Comment:
```suggestion
  React.useEffect(() => {
    if (!isFitSupported) {
      objectFitImages(ref.current);
    }
  }, []);

  state['aria-hidden'] = state.alt || state['aria-label'] ? undefined : 'true';
```

We can and should use just `.ref` 🤔 
_Do not forget to remove `imageRef` from `ImageState`_